ibm/service/power/resource_ibm_pi_workspace.go

```diff
@@ -121,6 +122,11 @@ func resourceIBMPIWorkspaceRead(ctx context.Context, d *schema.ResourceData, met
 		return diag.FromErr(err)
 	}
 	d.Set(PIWorkspaceName, controller.Name)
+	wsDetails := map[string]interface{}{
+		WorkspaceCreationDate: controller.CreatedAt,
+		WorkspaceCRN:          controller.CRN,
```

Comment:
This should be the targetcrn double check that when you ran it
```suggestion
		WorkspaceCRN:          controller.TargetCRN,
```